streamx-flink/streamx-flink-shims/streamx-flink-shims-base/src/main/scala/com/streamxhub/streamx/flink/core/SqlCommandParser.scala

```diff
@@ -311,178 +317,205 @@ object SqlCommand extends enumeratum.Enum[SqlCommand] {
    */
   case object ALTER_FUNCTION extends SqlCommand(
     "alter function",
-    "(ALTER\\s+(TEMPORARY\\s+|TEMPORARY\\s+SYSTEM\\s+|)FUNCTION\\s+.*)"
+    "(ALTER\\s+(TEMPORARY\\s+|TEMPORARY\\s+SYSTEM\\s+|)FUNCTION\\s+.+)"
   )
 
 
-  //---- INSERT Statement
+  //---- INSERT Statement--------------------------------------------------------------------------------------------------------------------------------
 
   /**
    * INSERT { INTO | OVERWRITE } [catalog_name.][db_name.]table_name [PARTITION part_spec] [column_list] select_statement
    * INSERT { INTO | OVERWRITE } [catalog_name.][db_name.]table_name VALUES values_row [, values_row ...]
    */
   case object INSERT extends SqlCommand(
     "insert",
-    "(INSERT\\s+(INTO|OVERWRITE)\\s+.*)"
+    "(INSERT\\s+(INTO|OVERWRITE)\\s+.+)"
   )
 
-  //---- DESCRIBE Statement
+  //---- DESCRIBE Statement--------------------------------------------------------------------------------------------------------------------------------
 
   /**
    * { DESCRIBE | DESC } [catalog_name.][db_name.]table_name
    */
   case object DESC extends SqlCommand(
     "desc",
-    "(DESC\\s+.*)"
+    "(DESC\\s+.+)"
   )
 
   /**
    * { DESCRIBE | DESC } [catalog_name.][db_name.]table_name
    */
   case object DESCRIBE extends SqlCommand(
     "describe",
-    "(DESCRIBE\\s+.*)"
+    "(DESCRIBE\\s+.+)"
   )
 
 
-  //---- DESCRIBE Statement
+  //---- EXPLAIN Statement--------------------------------------------------------------------------------------------------------------------------------
 
   /**
+   * For flink-1.13.x：EXPLAIN PLAN FOR `<query_statement_or_insert_statement>` <br>
+   * For flink-1.14.x：EXPLAIN ESTIMATED_COST, CHANGELOG_MODE, JSON_EXECUTION_PLAN `<query_statement_or_insert_statement>`<br>
+   * For flink-1.15.x：<br>
    * <pre>
-   * EXPLAIN PLAN FOR <query_statement_or_insert_statement>
+   * EXPLAIN [([ExplainDetail[, ExplainDetail]*]) | PLAN FOR] <query_statement_or_insert_statement_or_statement_set>
+   *
+   * statement_set:
+   * EXECUTE STATEMENT SET
+   * BEGIN
+   * insert_statement;
+   * ...
+   * insert_statement;
+   * END;
    * </pre>
+   * Recommended not to use the form of flink-1.15.x
    */
   case object EXPLAIN extends SqlCommand(
-    "explain plan for",
-    "(EXPLAIN\\s+PLAN\\s+FOR\\s+(SELECT\\s+.*|INSERT\\s+.*))"
+    "explain",
+    "(EXPLAIN\\s+.+)"
```

Comment:
It is not recommended to change it. The original definition is clearer

Reply:
flink-1.13.x：
EXPLAIN PLAN FOR <query_statement_or_insert_statement>
flink-1.14.x：
EXPLAIN [([ExplainDetail[, ExplainDetail]*]) | PLAN FOR] <query_statement_or_insert_statement>
    example: EXPLAIN ESTIMATED_COST, CHANGELOG_MODE, JSON_EXECUTION_PLAN SELECT ...